wiki/People/The_Team/th.md

```diff
@@ -1,56 +1,96 @@
+
+| ชื่อ | คำอธิบาย |
 | :-- | :-- |
-| [awp](https://osu.ppy.sh/users/2650) ([Barrel Roll Weekly](http://brw.twinkfish.com/)) | General housekeeping and help, ทำ artwork chibi ในเกม/เว็บไซต์ |
-| [Derekku](https://osu.ppy.sh/users/91341) | General housekeeping, จัดการเกี่ยวกับสังคมในเกม |
-| [Echo](https://osu.ppy.sh/users/431) ([blog](http://blog.echo.sh/)) | ทีม osu! dev, บูรณาการ IRC แชทของในเกม, ซ่อมบำรุงเว็บไซต์ |
-| [Intermezzo](https://osu.ppy.sh/users/136842) | ทีม osu! dev, osz2/p2p backend |
-| Jim ([Brand New Games](http://www.bravegamer.com/)) | ออกแบบเว็บไซต์ |
-| [LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](http://luigihann.deviantart.com/)) | ออกแบบ Epic skin, มีส่วนร่วมในการทำสกิน default และ iconset |
-| [mm201](https://osu.ppy.sh/users/30655) | ทีม osu! dev, mm sliders |
-| [nuudles](https://osu.ppy.sh/users/21312) | พัฒนา osu! iPhone |
-| Phil | ออกแบบฟอรัมเว็บ |
-| [Sarumaru](https://osu.ppy.sh/users/9427) ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | ต้นแบบแนวคิด pippi และออกแบบงาน artwork |
-| [Sinistro](https://osu.ppy.sh/users/5530) | รายงานการประชุม, มีส่วนร่วมใน FAQ |
-| [woc2006](https://osu.ppy.sh/users/1105845) | ทีม osu! dev, พัฒนาโหมด osu!mania |
-
-## กลุ่มของผู้ใช้งาน
-
-| ชื่อ | ลักษณะ |
+| [ผู้เสนอบีทแมพ](Beatmap_Nominators) | เป็นผู้ใช้งานที่ช่วยให้บีทแมพของคุณได้สถานะ qualified |
+| [ผู้พัฒนา](Developers) | ทำให้เกมดีขึ้นโดยการเพิ่มสิ่งใหม่ ๆ และแก้ไขบัค |
+| [ทีมผู้ดูแล](Global_Moderation_Team) | คอยดูแลทั้งในฟอรัมและแชทในเกม |
+| [ผู้ดูแลการเสนอ](Nomination_Assessment_Team) | ดูแลจัดการผู้เสนอบีทแมพ
+| [osu! alumni](osu!_Alumni) | พวกที่เคยเป็นผู้ร่วมทำให้เกมดีขึ้นมาก่อน |
+| [ทีม Project Loved](Project_Loved_Team) | ผู้รับรู้บีทแมพที่ชุมชนรักมากที่สุด |
+| [Support Team](Support_Team) | ให้ความช่วยเหลือสิ่งที่ขอในช่วยเหลือ/ฟีเจอร์ |
+
+## ทีมงานที่ไม่ได้ใช้งาน (inactive)
+
+| ชื่อ | บทบาท |
 | :-- | :-- |
-| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | คอยดูแลทั้งในฟอรัมและแชทในเกม |
-| [Beatmap Nomination Group](/wiki/People/Beatmap_Nomination_Group) | เป็นผู้ใช้งานที่คอยช่วยให้บีทแมพได้ qualified |
-| [Developers](https://osu.ppy.sh/groups/11) | ทำให้เกมดีขึ้นโดยการเพิ่มสิ่งใหม่ๆและแก้ไขบัคในเกม |
-| [osu! alumni](/wiki/People/osu!_Alumni) | พวกที่เคยเป็นผู้ร่วมพัฒนาเกมมาก่อน |
-| [Support Team Redux](/wiki/People/Support_Team) | ให้ความช่วยเหลือสิ่งที่ขอใน ช่วยเหลือ/ลูกเล่น |
-| [osu!news](https://osu.ppy.sh/groups/25) | เขียนข่าวเกี่ยวกับ osu! ตั้งแต่ปี 2013 |
-| [Tournament Staff](https://osu.ppy.sh/groups/26) | Responsible for handling officially supported osu! tournaments across all gamemodes. |
-| [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) | ช่วยแมพของคุณให้มีคุณภาพที่ดีขึ้น |
+| ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) | ทำหน้าที่และช่วยเหลือทั่วไป, ทำ งานศิลปิน chibi. ([Barrel Roll Weekly](http://brw.twinkfish.com/)) |
```

Comment:
Normally Thai does not use full stops, but required here as to distinguish between the sentence and the link.

Reply:
```suggestion
| ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) | ทำหน้าที่และช่วยเหลือทั่วไป, ทำงานศิลปิน chibi. ([Barrel Roll Weekly](http://brw.twinkfish.com/)) |
```